doc/release-notes/8.3.md

```diff
@@ -0,0 +1,30 @@
+# Release notes for Gluster 8.3
+
+This is a bugfix release. The release notes for [8.0](8.0.md), [8.1](8.1.md) and [8.2](8.2.md)
+contain a listing of all the new features that were added
+and bugs fixed in the GlusterFS 8 stable release.
+
+**NOTE:** Next minor release tentative date: Week of 20th Feb, 2021
+
+## Builds are available at 
+
+[https://download.gluster.org/pub/gluster/glusterfs/8/8.3/](https://download.gluster.org/pub/gluster/glusterfs/8/8.3/)
+
+## Issues addressed in this release
+
+Please find the list of issues added to this release below.
+
+- [#1000](https://github.com/gluster/glusterfs/issues/1000) [bug:1193929] GlusterFS can be improved
```

Comment:
This and the next have no value - especially not at the top. If you are sorting the bugs, I suggest from newest to oldest.